lazybytez/jojo-discord-bot
# Add statistics overview

## Description

* [x] Uptime
* [x] Memory Consumption
* [x] Number running Goroutines ("Tasks" or so)
* [x] Connected Servers
* [x] Cluster Identifier (optional)

## Related issue

Resolves  #48

## How can this be tested?

Execute it, play with the bot ping command, execute it and see if something changed. If yes, then it works.
